terms/roccurves_pb.tex

```diff
@@ -0,0 +1,88 @@
+
+The diagonal line from (0,0) to (1,1) characterises a test that cannot distinguish between diseased and non-diseased states, that is, a random guess. This line often serves as a benchmark for judging how well (or bad) a test performs.
+
+\begin{figure}
+	\includegraphics{roc_curves/Figure1.png}
+	\caption{Basic ROC plot with five classifiers.}
+	\label{fig:classifiers-roc}
+\end{figure}
+
+\section{Evaluating Classifiers} \index{roc-curves!evaluating classifiers}
+The best possible prediction method would yield a point in the upper left corner of the ROC space representing 100\% sensitivity and 100\% specificity, i.e. a perfect classifier. On the other hand, a random guess would outcome a point along or close to the diagonal line. Classifiers A and B yield good classification results whilst points below the line represent bad results (worse than random). The ROC plot allows us to compare classifiers and choose the one that is closest to (0,1) and furthest from TPR equals FPR (the diagonal line).
+
+Consider now that the blood protein levels in diseased people and healthy people are normally distributed. A medical test might measure the level of a certain protein in a blood sample and classify any number above a certain threshold as indicating disease. Thus given a threshold parameter $p$, the instance is labelled positive if the continuous random variable $X$ is greater than $p$ and negative otherwise. As shown in figure 2 (left), choosing a high threshold produces a low likelihood of a false positive (while increasing the specificity) and a high likelihood of a false negative result (while decreasing sensitivity). Choosing a low threshold results in the opposite effect. Thus a different threshold would result in a different sensitivity and specificity pair. By repeating the experiment with adjusted thresholds different values for FPR and TPR are obtained leading to different positioning in the ROC space. The classifier that is able to increase the rate of detection while keeping the false alarm rate low is considered a good classifier. Figure 2 (right) shows the ROC curves with different thresholds. As the ROC curve gets closer to the top left hand of the graph the more optimal the threshold and the better the ability to discriminate between the classes.
```

Comment:
Are you serious???? This is a sure way how to fail your MSc and end up in front of a disciplinary board.  This example is taken verbatim from https://en.wikipedia.org/wiki/Receiver_operating_characteristic.  Absolutely UNACCEPTABLE.